src/garage/experiment/local_runner.py

```diff
@@ -1,69 +1,24 @@
-"""
-The local runner for tensorflow algorithms.
-
-A runner setup context for algorithms during initialization and
-pipelines data between sampler and algorithm during training.
-"""
 import copy
 import time
 import types
 
 from dowel import logger, tabular
-import tensorflow as tf
 
 from garage.experiment.snapshotter import Snapshotter
 
-# Note: Optional module should be imported ad hoc to break circular dependency.
-
 
 class LocalRunner:
-    """This class implements a local runner for tensorflow algorithms.
-
-    A local runner provides a default tensorflow session using python context.
-    This is useful for those experiment components (e.g. policy) that require a
-    tensorflow session during construction.
-
-    Use Runner.setup(algo, env) to setup algorithm and environement for runner
-    and Runner.train() to start training.
+    """Base class of local runner.
```

Comment:
I feel like this docstring should provide more context of what the runner is for.
Specifically, that the runner is the interface which controls the lifecycle of the algorithm, and manages services within that lifecycle (specifically, snapshotting, resuming, and sampling).

Reply:
Yes -- please provide a nice long description. this is a major interface.